src/gen_dragonfruit/gd_board.cpp

```diff
@@ -63,7 +63,7 @@ static void gd_board_print_build_opts()
 static void gd_board_setup(struct gd_board* b){
     Serial.begin(9600);
     Serial.println(F("Board Setup Start"));
-
+    digitalWrite(_PIN_SEN_EN_, HIGH);
```

Comment:
Since this function isn't as explicit as the open() functions for the sensors, it might be good to comment this particular line and describe why it is called.